ankdown/ankdown.py

```diff
@@ -46,10 +46,15 @@
     --css CSS_STRING   CSS Config as String
 
     --dollar    Uses dollar sign as math separator instead of brackets
+
+    --config CONFIG_STRING   ankdown configuration as json string
+
+    --configFile CONFIG_FILE_PATH path to ankdown configuration as json file
 """
 
 
 import hashlib
+import json
```

Comment:
Did you consider using yaml? In general I think it's a slightly better choice for config files, since it supports comments.

Reply:
I had not thought of that. I am currently working a lot in the JavaScript Cosmos. There is JSON standard. I'll take a look at YAML.

Reply:
It's actually a superset of JSON, so any config you wanted to pass would still be valid